applications/RomApplication/tests/NonlinearTestFiles/MainKratosROM.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics
+import KratosMultiphysics.ConvectionDiffusionApplication
+import KratosMultiphysics.RomApplication as romapp
+
+from KratosMultiphysics.assign_scalar_variable_process import AssignScalarVariableProcess
+from KratosMultiphysics.ConvectionDiffusionApplication.apply_thermal_face_process import ApplyThermalFaceProcess
+from KratosMultiphysics.ConvectionDiffusionApplication.convection_diffusion_analysis import ConvectionDiffusionAnalysis
+
+import h5py
+import json
+import sys
+import time
+
+class ConvectionDiffusionAnalysisWithFlush(ConvectionDiffusionAnalysis):
+
+    def __init__(self,model,project_parameters,flush_frequency=10.0):
+        super(ConvectionDiffusionAnalysisWithFlush,self).__init__(model,project_parameters)
+        self.flush_frequency = flush_frequency
+        self.last_flush = time.time()
+
+    def _CreateSolver(self):
+        print(sys.path, file=sys.stderr)
+        print(self.parallel_type)
+        if self.parallel_type == "OpenMP":
+            from KratosMultiphysics.RomApplication import ConvectionDiffusionROMsolver
+            return ConvectionDiffusionROMsolver.ROMSolver(self.model, self.project_parameters["solver_settings"])
+        else:
+            import trilinos_static_solver

```

Comment:
I think that the ROM application does not work in MPI yet. If this is the case, please remove this.